apps/ledger-live-desktop/src/renderer/screens/market/MarketList.tsx

```diff
@@ -219,6 +220,10 @@ const CurrencyRow = memo(function CurrencyRowItem({
   const isStarred = currency && starredMarketCoins.includes(currency.id);
   const availableOnBuy = currency && onRampAvailableTickers.includes(currency.ticker.toUpperCase());
   const availableOnSwap = internalCurrency && swapAvailableIds.includes(internalCurrency.id);
+  const stakeProgramsFeatureFlag = useFeature("stakePrograms");
+  const { params: paramsFlag, enabled: stakeProgramsEnabled } = stakeProgramsFeatureFlag || {};
```

Comment:
Is typescript passing this code? `listFlag` could be undefined but we're not checking for it on line 226. I think we'll have better type safety without the destructuring, so something like this instead:

```
const listFlag = stakeProgramsFeatureFlag?.params?.list ?? []
const stakeProgramsEnabled = stakeProgramsFeatureFlag?.enabled
```